vasodecanto-app/vasodecanto-app/DesignSystem/Fonts/AppFont.swift

```diff
@@ -0,0 +1,43 @@
+//  AppFont.swift
+//  vasodecanto-app
+//
+//  Created by Jade Oliveira on 29/09/25.
+
+import SwiftUI
+
+// MARK: - Estilos de Fonte do Aplicativo
+struct AppFont {
+    // MARK: – Estilos Customizados Heebo
+    /// - Fonte: **14.0**
+    static func heeboCaption() -> Font {
```

Comment:
Sugestão: Como está sendo utilizado funções sem parâmetros porque já se tem valores definidos para tamanho e fonte, seria interessante tornar essas funções em variáves comuns ou até mesmo variáveis computadas. Melhora a legibilidade deixa o uso ainda mais limpo. Exemplo:

```suggestion
    static var heeboCaption: Font {
        .custom("Heebo-Bold", size: FontSize.caption)
       }
```